modules/ROOT/pages/shared-resources.adoc

```diff
@@ -150,11 +150,29 @@ Example:
 
 === Referencing Shared Resources
 
-In the following example, `mule-domain-config.xml`, an HTTP connector is defined as a shared resource.
+In the following example, `mule-domain-config.xml`, an HTTP connector listener configuration is defined as a shared resource as well as an HTTP connector requester configuration.
```

Comment:
suggest rewording a bit to something like this:
The following `mule-domain-config.xml` example defines HTTP listener and HTTP request configurations as shared resources.

or this (but seems like the subsequent paragraph covers it):
The following `mule-domain-config.xml` example defines HTTP listener and HTTP request configurations as shared resources that HTTP connectors in different projects can use.

Reply:
I like the first option better. Feel free to change it accordingly.